virtme_ng/version.py

```diff
@@ -47,7 +47,7 @@ def get_version_string():
             version = version[1:]
 
         # Replace hyphens with plus sign for build metadata
-        version_pep440 = version.replace("-", "+", 1).replace("-", ".")
+        version_pep440 = '0.0+dev'+version.replace("-", "+").replace("+", ".")
```

Comment:
@likewhatevs @arighi : is this `'0.0+dev'+` expected to be there all the time?

```
./vng -V
virtme-ng 0.0+dev1.29.5.ge031324
```

In @likewhatevs example from the PR, the version is different:

```
0.0+dev27b7bc8
```

Is it because your git repo doesn't contain any (annotated) tags?

Maybe this `0.0+dev` should only be appended if the version doesn't start with `v`?
Or in this case, a fallback to `PKG_VERSION` should be done? → Then we can drop the `--always` from `git describe`

Reply:
That was only intended to be there when fallback to commit hash happened, thanks for fixing that!